app/models/snippet.rb

```diff
@@ -1,5 +1,6 @@
 class Snippet < ApplicationRecord
   belongs_to :user
+  validates :name, :artist, :notes, presence: true
```

Comment:
True, I think it makes more sense as part of this PR: [Add snippet feature spec #5](https://github.com/entcheva/music_snippets/pull/4)